common/api/imodeljs-editor-frontend.api.md

```diff
@@ -87,8 +88,6 @@ export class CreateArcTool extends CreateOrContinuePathTool {
     // (undocumented)
     protected getArcNormal(ev: BeButtonEvent): Vector3d;
     // (undocumented)
-    protected _getMethodDescription: () => PropertyDescription;

```

Comment:
Need to revert/re-run extract api after reverting changes to SketchTool.ts? Shouldn't be any changes to editor package right?

Reply:
> Need to revert/re-run extract api after reverting changes to SketchTool.ts? Shouldn't be any changes to editor package right?

@bbastings - I just fixed, I forgot to clean and rebuild after reverting SketchTool.ts